index.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
+    <script
+      src="https://kit.fontawesome.com/8b2919a184.js"
+      crossorigin="anonymous"
+    ></script>
+    <title>Mobile Portfolio</title>
+  </head>
+  <body>
+    <div class="main-container parent-div">
+      <div class="centered-div">
+        <nav class="nav-container">
+          <div class="logo">
+            <span>My Logo</span>
+          </div>
+          <div class="menu-bars">
+            <i class="fa-solid fa-bars"></i>
+          </div>
+        </nav>
+        <main>
+          <div class="content-container">
+            <div class="heading-text">
+              <h2>I'm Marine</h2>
+              <h2>Glad to see you!</h2>
+            </div>
+            <div class="decription-text text">
+              <p>
+                I’m a software developer! I can help you build a product ,
+                feature or website Look through some of my work and experience!
+                If you like what you see and have a project you need coded,
+                don’t hestiate to contact me.
+              </p>
+            </div>
+            <div class="message-text text">
+              <p>LET’S CONNECT</p>
+            </div>
+            <div class="icons-container">
+              <div class="child-container">
+                <span
+                  ><img
+                    class="img-twitter"
+                    src="images/twitter.png"
+                    alt="twitter"
+                /></span>
+                <span
+                  ><img class="img-i" src="images/I.png" alt="linked in" />
+                  <img class="img-N" src="images/N.png" alt="linked in"
+                /></span>
+                <span
+                  ><img class="img-m" src="images/M.png" alt="Messanger"
+                /></span>
+                <span
+                  ><img class="img-cat" src="images/cat.png" alt="cat icon"
+                /></span>
+                <span
+                  ><img
+                    class="img-thumbs"
+                    src="images/thumps.png"
+                    alt="cat icon"
+                /></span>
+              </div>
```

Comment:
- [ ] Kindly make sure to wrap all social media into the `ul` tag and make make sure each is wrapped into the `li` tag.
- [ ] Kindly make sure to align the contents using the CSS `flexbox` because it gives us a lot of options to align all our elements, plus, it's responsive.